### DISCUSSION TITLE
Sort나 Join에 왜 버퍼가 따로 필요할까?

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#13] [Q&A] [answered]
로컬 메모리 영역에서 "쿼리의 용도별로 필요할 때만 공간이 할당"이라는 문장이 있었는데요.
이 때 예시로 sort buffer와 join buffer가 언급되었습니다.

다른 연산이 아닌 왜 이 두 연산에서만 공간이 할당될까요?

**Comment by menthamin** [comment, marked as answer]
조인버퍼, 소트 버퍼 사이즈가 각각 있다.
버퍼사이즈로 감당이 안되면 결국 디스크 I/O가 발생
